mam/api/api.c

```diff
@@ -102,6 +102,34 @@ static mam_channel_t *mam_api_get_channel(mam_api_t const *const api,
   return NULL;
 }
 
+mam_endpoint_t *mam_api_get_endpoint(mam_api_t const *const api,
```

Comment:
Do you think the overhead of doing this without the associated `channel_id ` is  too big ?

Reply:
obviously depends on the number of channels, but there's no reason why user should not be able to provide the channel